### DISCUSSION TITLE
@some-alias.all isn't working after upgrading to v6

### DISCUSSION THREAD

**Discussion by gilm123** [cypress-io/cypress#14174] [Questions and Help]
hi, after upgrading to cypress 6.1.0, i noticed all of my tests that check intercept aliases with `.all` are failing.
I'm able to see the xhr calls both at the "ROUTES" expander in the cypress window, and in the dev-tools network window, but when I'm querying the same alias with `.all`,
for example, `cy.get("@some-alias.all")` it always returns undefined.
what am i missing ? it used to work at v5

**Comment by jennifer-shehane** [comment]
This will be supported in our next release. This syntax was fixed in this PR specifically: https://github.com/cypress-io/cypress/pull/9594

**Reply by gilm123** [reply]
> This will be supported in our next release. This syntax was fixed in this PR specifically: #9594

thank you,
i wonder if i should downgrade or wait for the next release,
do you have any time estimation?

**Comment by bahmutov** [comment]
We hope to release on Monday Dec 21st
